config/sync/language/da/views.view.files.yml

```diff
@@ -26,8 +26,6 @@ display:
           label: Ændringsdato
         count:
           label: 'Brugt i'
-          alter:
-            path: 'admin/content/files/usage/{{ fid }}'
```

Comment:
This is also a thing i keep seeing repeated in config. 
I think you mentioned before you wanted to talk about this, but i dont think we ever did.